k9mail/src/test/java/com/fsck/k9/crypto/MessageDecryptVerifierTest.java

```diff
@@ -298,8 +300,8 @@ public static Part getPart(Part searchRootPart, int... indexes) {
     }
 
     //TODO: Find a cleaner way to do this
-    private static void addProtocolParameter(Part part) throws MessagingException {
-        String contentType = part.getContentType();
-        part.setHeader(MimeHeader.HEADER_CONTENT_TYPE, contentType + "; protocol=\"application/pgp-encrypted\"");
+    private static void setContentTypeWithProtocol(Part part, String mimeType, String protocol)
```

Comment:
To make it obvious inside the tests how the parts are structured. `addProtocolParameter` does not exactly imply "with application/pgp-encrypted protocol to content type header", the calls as they are now show exactly what they do